lms/static/sass/shared-v2/_components.scss

```diff
@@ -150,6 +150,13 @@
   }
 }
 
+.section {
+    .icon {
+        width: 20px;
+        text-align: center;
+    }
+}
+
```

Comment:
The sections are actually on the course page, not the updates. I had to do this to accommodate a Font Awesome icon that is wider than normal:

![image](https://cloud.githubusercontent.com/assets/5985072/25402800/e1d4b35c-29c7-11e7-8c93-b666ca1a1924.png)

At @marcotuts's suggestion, I'm center aligning and padding the icon so that the text can left align.